package.json

```diff
@@ -43,18 +43,23 @@
     "@types/validator": "^13.7.1",
     "debug": "^4.3.3",
     "dottie": "^2.0.2",
+    "gcc": "^1.0.2",
```

Comment:
why these additions and changes in exisiting packages in package.json? Are they required for yugabytedb dialect to work.

Reply:
These are the changes the `npm install` made to the `package.json` while I faced few issues in installation in the initial step. @sanyamsinghal  Shall I not add them ?

Reply:
@priyanshi-yb  i am okay with the new packages added here.
But i am a bit skeptical whether main sequelize repo will allow these changes like:
"ibm_db": "2.8.1" -> "ibm_db": "^2.8.1"
and similar at line 106, 120
Are these mandatory? what does '^' sign means here?

Reply:
it is way to check version as ^1.x = ( >=1.x && <=2.0 ).

Reply:
Not mandatory removed them.